src/main/java/tuteez/ui/PersonListPanel.java

```diff
@@ -27,23 +34,86 @@ public PersonListPanel(ObservableList<Person> personList) {
         super(FXML);
         personListView.setItems(personList);
         personListView.setCellFactory(listView -> new PersonListViewCell());
+        startCentralRefreshTimeline();
     }
 
+    /**
+     * Starts the central timeline that refreshes all visible PersonCards every minute.
+     */
+    private void startCentralRefreshTimeline() {
+        if (centralRefreshTimeline != null) {
+            centralRefreshTimeline.stop();
```

Comment:
Can i confirm that if this code reaches this point is there supposed to be some bug? because somehow this method is called more than once per panel. If that was the case maybe you write an assert statement here or log it just in case. 

Additionally perhaps it is better to not `stop()` but `return`? Do you expect behaviour of the earlier `centralRefreshTime` to be different till where you need to reset everything?